.storybook/main.js

```diff
@@ -1,46 +1,46 @@
 module.exports = {
   stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: [
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
-    "@storybook/addon-links",
-    // React 18 causes some issues with storyshots. In package.json we have a resolutions field for
-    // react-test-renderer to get around this. (https://github.com/storybookjs/storybook/issues/17985)
-    "@storybook/addon-storyshots",
-  ],
-  framework: "@storybook/react",
-  core: {
-    builder: "@storybook/builder-vite",
+  addons: ["@storybook/addon-essentials", "@storybook/addon-interactions", "@storybook/addon-links",
+  // React 18 causes some issues with storyshots. In package.json we have a resolutions field for
+  // react-test-renderer to get around this. (https://github.com/storybookjs/storybook/issues/17985)
+  "@storybook/addon-storyshots"],
+  framework: {
+    name: "@storybook/react-vite",
+    options: {}
   },
   features: {
     // storyStoreV7 is enabled by default and works well for our general Storybook configuration.
     // However, we have to disable it because it breaks storyshots. Storybook developers are working
     // on a new replacement addon for storyshots to fix this in the future.
     // (https://github.com/storybookjs/storybook/issues/15916)
-    storyStoreV7: false,
+    storyStoreV7: false
   },
   typescript: {
     reactDocgen: "react-docgen-typescript",
     reactDocgenTypescriptOptions: {
       compilerOptions: {
         allowSyntheticDefaultImports: false,
-        esModuleInterop: false,
-      },
-    },
+        esModuleInterop: false
```

Comment:
This is a nit/partially my own curiosity, but it seems like now we're applying different lint rules in this file than elsewhere in the repo (i.e. no trailing comma in an object like this). It'd be nice to have everything be standardized if you can figure out why our linter is being overriden.

Reply:
Good catch i'll take a look at the linter rules

Reply:
https://stackoverflow.com/questions/57947585/eslint-warning-file-ignored-by-default-use-a-negated-ignore-pattern

Looks like eslint ignores files that start with `.` by default and you have to intentionally negate the ignored file.